hack/run-ci-e2e-test.sh

```diff
@@ -103,6 +103,9 @@ OSDK_WMCO_test $OSDK "-run=TestWMCO/operator_deployed_without_private_key_secret
 # Run the creation tests of the Windows VMs
 OSDK_WMCO_test $OSDK "-run=TestWMCO/create -v -timeout=90m -node-count=$NODE_COUNT --private-key-path=$KUBE_SSH_KEY_PATH $WMCO_PATH_OPTION"
 
+# Run the upgrade tests and skip deletion of the Windows VMs
```

Comment:
Please call `get_WMCO_logs` after running the creation test suite or we will lose the logs from the first pod.